cmd/sales-api/main.go

```diff
@@ -58,10 +58,7 @@ func main() {
 			WriteTimeout    time.Duration `default:"5s" envconfig:"WRITE_TIMEOUT"`
 			ShutdownTimeout time.Duration `default:"5s" envconfig:"SHUTDOWN_TIMEOUT"`
 		}
-		DB struct {
-			DialTimeout time.Duration `default:"5s" envconfig:"DIAL_TIMEOUT"`
-			Host        string        `default:"mongo:27017/gotraining" envconfig:"HOST"`
-		}
+		DB    database.Config
```

Comment:
Not sure I like this at all. I rather see it and the tags. This is hiding information. Let's not couple these things together.